cypress/integration/userCanChooseFood.feature.js

```diff
@@ -0,0 +1,35 @@
+describe("Restaurants", () => {
+  it("needs a budget input", () => {
+    const nativeInputValueSetter = Object.getOwnPropertyDescriptor(
+      window.HTMLInputElement.prototype,
+      "value"
+    ).set;
+    const changeRangeInputValue = $range => value => {
+      nativeInputValueSetter.call($range[0], value);
+      $range[0].dispatchEvent(new Event("change", { value, bubbles: true }));
+    };
```

Comment:
Hmmm, I guess I can sort of figure out what this does...
Can you take a few min and see if you can solve it with cypress trigger() instead?
https://docs.cypress.io/api/commands/trigger.html#Change-Event

Reply:
I have spent a fair share of time with that. This is the official workaround.
[](https://github.com/cypress-io/cypress/issues/1570)

Reply:
OK!